registry/server/templates/routes/getTemplates.ts

```diff
@@ -2,12 +2,19 @@ import { Request, Response } from 'express';
 
 import db from '../../db';
 import Template from '../interfaces';
+import { tables } from '../../db/structure'
+import { appendDigest } from '../../util/hmac';
+import { EntityTypes } from '../../versioning/interfaces';
 
 const getTemplates = async (req: Request, res: Response): Promise<void> => {
-    const templates = await db.select().from<Template>('templates');
+    const templates = await db
+        .selectVersionedRowsFrom<Template>(tables.templates, 'name', EntityTypes.templates, [`${tables.templates}.*`]);
+    const itemsWithId = templates.map(item => {
```

Comment:
what about marking this map as a part of selectVersionedRowsFrom or moving to some util function?